osu.Game/Localisation/CommonStrings.cs

```diff
@@ -54,6 +54,11 @@ public static class CommonStrings
         /// </summary>
         public static LocalisableString Downloading => new TranslatableString(getKey(@"downloading"), @"Downloading...");
 
+        /// <summary>
+        /// "You have downloaded too many {0}s! Please try again later."
+        /// </summary>
+        public static LocalisableString TooManyDownloaded(string humanisedModelName) => new TranslatableString(getKey(@"too_many_downloaded"), @"You have downloaded too many {0}s! Please try again later.", humanisedModelName);
```

Comment:
I'm just going to offhand mention that there is no way this is going to localise well to any language...

* The string is accepting the humanised model name, which currently is only ever English.
* The non-parameterised string content of this localisable is hard-coding English pluralisation rules. This is not going to scale to any language that has even moderately complex inflection rules.

At the point where a localisation is going to have this many issues I honestly think it's better to leave hardcoded English rather than have to live with the consequences of what this jerry-rigged setup may bring...

Reply:
Right... will remove it for now, can be reiterated later on.

Reply:
Adding the entire sentence (i.e. put in "beatmaps" inside the localisable string rather than jam it through using a parameter) also would work for now, I guess. I'm generally not sure you can ever realistically expect to be able to drop in a plural noun into a localisation like that and have it work in all languages.